client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/jetpack-search.tsx

```diff
@@ -0,0 +1,63 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { translate } from 'i18n-calypso';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import ThankYouJetpackSearchProduct from '../products/jetpack-search-product';
+
+export type JetpackSearchThankYouProps = {
+	purchase: ReceiptPurchase;
+};
+
+export default function JetpackSearchThankYou( { purchase }: ThankYouJetpackSearchProductProps ) {
+	const siteId = useSelector( getSelectedSiteId );
+	const footerDetails = [
+		{
+			key: 'footer-generic-support',
+			title: translate( 'Everything you need to know' ),
+			description: translate( 'Explore our support guides and find an answer to every question.' ),
+			buttonText: translate( 'Explore support resources' ),
+			buttonHref: localizeUrl( 'https://wordpress.com/support/' ),
+			buttonOnClick: () => {
+				recordTracksEvent( 'calypso_thank_you_footer_link_click', {
+					context: 'jetpack-search',
+					type: 'generic-support',
+				} );
+			},
+		},
+		{
+			key: 'footer-plugins-support',
```

Comment:
@chad1008 A heads up that I also added a second footer blurb (copy taken from the marketplace Congrats page: https://github.com/Automattic/wp-calypso/blob/b60cadaf8602ef05ea2f39d6712baa6a840d5bab/client/my-sites/checkout/checkout-thank-you/marketplace/use-thank-you-footer.tsx#L72-L78), thought it'd be appropriate to also provide plugin information, given that the user just installed the Jetpack Search plugin.